cloudformation.yaml

```diff
@@ -743,6 +748,86 @@ Resources:
       Principal: "events.amazonaws.com"
       SourceArn: !GetAtt BillingLambdaDailyRule.Arn
 
+
+  # Watch API Renewer Function
+  LambdaApiWatchRenewerFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      PackageType: Image
+      ImageUri: !Ref ApiWatchRenewerImageUri
+      Role: !GetAtt LambdaApiWatchRenewerFunctionExecutionRole.Arn
+      FunctionName: !Sub "${Environment}-sheetsapi-api-watch-renewer-lambda"
+      Timeout: 30
+      Environment:
+        Variables:
+          GOOGLE_CLIENT_ID: !Ref GoogleClientId
+          GOOGLE_CLIENT_SECRET: !Ref GoogleClientSecret
+          OAUTH_SECRET_TOKEN: !Ref OAuthSecretToken
+          ENVIRONMENT: !Ref Environment
+          SHEETS_API_TABLE: !Ref DynamoDBTableV2
+          SENTRY_DSN: !Ref SentryDSN
+          API_BASE_URL: !Ref ApiBaseUrl
+          CLIENT_BASE_URL: !Ref ClientBaseUrl
+          CLIENT_APP_BASE_URL: !Ref ClientAppBaseUrl
+          STRIPE_SECRET_KEY: !Ref StripeSecretKey
+          STRIPE_WEBHOOK_SECRET: !Ref StripeWebhookSecret
+          STRIPE_SUBSCRIPTION_PRICE_ID: !Ref StripeSubscriptionPriceId
+          STRIPE_USAGE_BASED_PRICE_ID: !Ref StripeUsageBasedPriceId
+          CLOUDFRONT_DISTRIBUTION_ID: !Ref CloudFrontDistributionId
+          COOKIE_ALLOWED_DOMAIN: !Ref CookieAllowedDomain
+          ENCRYPTION_KEY_ID: !Ref EncryptionKey
+          EMAILS_ENABLED: "true"
+          SES_CONFIGURATION_SET: !Ref SESConfigurationSet
+          SES_DEFAULT_FROM_EMAIL: "hello@jedwal.co"
+          WEBHOOK_QUEUE_URL: !Ref WebhookQueue
+          IMAGE_STORAGE_BUCKET_URL: !Sub "https://${ImageStorageBucket}.s3.${AWS::Region}.amazonaws.com"
+          IMAGE_STORAGE_BUCKET: !Ref ImageStorageBucket
+
```

Comment:
Really annoying that we need to repeat this env setup every time. Maybe there's a better way?